### ISSUE TITLE
not so useful output

### ISSUE THREAD

**Issue by gforcada** [pypa/readme_renderer#29]
My setup.py looks like this:

```
long_description = """{0}
{1}


LICENSE
=======

{2}
""".format(
    open('README.rst').read(),
    open('CHANGES.rst').read(),
    open(os.path.join('docs', 'LICENSE.txt')).read(),
)

setup(
    ...
    long_description=long_description,
    ...
)
```

Running `python setup.py check -r -s` reports two errors:

```
running check
warning: check: Possible incomplete section title.
Treating the overline as ordinary text because it's so short. (line 573)

warning: check: Possible incomplete section title.
Treating the overline as ordinary text because it's so short. (line 573)

error: Please correct your package.
```

The problem is that saying that line 573 is wrong is not so useful as there are three files involved on the long_description (`README.rst`, `CHANGES.rst` and `docs/LICENSE.txt`).

It would be so much more useful if the offending line(s) are shown, so searching for the error would be much more clear, in my case that was this header:

```
???

---
```

**Comment by JulienPalard** [comment]
I'm able to reproduce a similar error with:

```
ABC 123
------
```

But the error is not emmited by `readme_renderer` but (docutils){https://sourceforge.net/projects/docutils/].

**Comment by jonparrott** [comment]
I believe we fixed this a while back, but #92 should completely close the loop on human-readable error messages.
